samples/openthread/coprocessor/README.rst

```diff
@@ -117,15 +118,22 @@ Building and running
 
 .. _ot_coprocessor_testing:
 
+HCI support
```

Comment:
Could you please extend this a bit explaining the use case for this feature? Also we might want to extend Thread architectures overview page: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/protocols/thread/overview/architectures.html#radio-co-processor-rcp

Reply:
Jira ticket KRKNWK-18076